lib/media/streaming_engine.js

```diff
@@ -3080,6 +3086,33 @@ shaka.media.StreamingEngine = class {
       return;
     }
 
+    if (error.code == shaka.util.Error.Code.SEGMENT_MISSING) {
+      const presentationTime = this.playerInterface_.getPresentationTime();
```

Comment:
As `segmentReferenceNeeded_()` takes gaps into account, do we still need this case here?